api/lib/notification_parser.js

```diff
@@ -7,18 +9,22 @@ var ripple = require('ripple-lib');
  *
  * @param {Ripple Transaction in JSON Format} notification_details.transaction
  * @param {RippleAddress} notification_details.account
- * @param {Hex-encoded String|ResourceId} notification_details.previous_transaction_identifier
- * @param {Hex-encoded String|ResourceId} notification_details.next_transaction_identifier
+ * @param {Hex-encoded String|ResourceId}
+ *                    notification_details.previous_transaction_identifier
```

Comment:
I'm not sure, I disabled jsdoc validation since they will need to be updated after the new interface is finished.